panther_lights/README.md

```diff
@@ -4,22 +4,44 @@ Package used to control the Husarion Panther LED panels.
 
 ## ROS Nodes
 
+### driver_node.py
+
+Node responsible for displaying frames on the Husarion Panther robot LED panels.
+
+#### Subscribe
+
+- `panther/lights/driver/front_panel_frame` [*sensor_msgs/Image*]: an animation frame to be displayed on robot front LED panel. 
+- `panther/lights/driver/rear_panel_frame` [*sensor_msgs/Image*]: an animation frame to be displayed on robot rear LED panel.
```

Comment:
I think it will be nice if there will be information about the size of the image. Like 1x46 pixels.